cadc-vos-client/src/main/java/org/opencadc/vospace/client/Main.java

```diff
@@ -615,29 +611,25 @@ private String getContentLength(Node n, boolean simple) {
 
     private void copyToVOSpace()
         throws Throwable {
-        URI originalDestination = null;
-        if (StringUtil.hasText(destination.getQuery())) {
-            originalDestination = new URI(destination.toString());
-            destination = new URI(destination.toString().replace("?" + destination.getQuery(), ""));
-        }
-
-        View view = null;
-        if (originalDestination != null) {
-            view = createAcceptsView(new VOSURI(originalDestination), null);
-        }
-        if (view == null) {
-            view = new View(VOS.VIEW_DEFAULT);
-        }
+        //URI originalDestination = null;
```

Comment:
I think a TODO is missing here.

Reply:
yeah, also just doing enough to get it to work for basics so commenting out extra features... it still needs a lot of work and wasn't even included in the branch CI before